### ISSUE TITLE
NEMS with pihole (help required)(not bug or maybe)

### ISSUE THREAD

**Issue by Decapitater** [pi-hole/pi-hole#4460]
### Versions

- Pi-hole:  latest
- AdminLTE:  latest
- FTL:  latest

### Platform

- OS and version:  NEMS 

- Platform:  RpI 3b+

### Expected behavior
Expected to work out of the box , after adding it as dns in router
_A clear and concise description of what you expected to happen._

### Actual behavior / bug
When typed the ip of Pi it's bound to NEMS , when used with ip/admin it opens pi hole webpage. It does not block when using it with router but it does when used individually. 
_A clear and concise description of what the bug is._

### Steps to reproduce

1. Setup NEMS linux on rpi with static ip
2. Download and install pi hole 
3. Allow it to use static ip of the NEMS 
4. Setup pi hole with router 

## Debug Token

- URL: https://tricorder.pi-hole.net/6GNMBQXA

### Screenshots

_If applicable, add screenshots to help explain your problem._

### Additional context

_Add any other context about the problem here._
Tldr : I have NEMS linux on with a static ip , setted up pi hole , pi hole page appears at ip_address/admin , NEMS page appears at ip_address 
Ip given = 192.168.29.254

Help required , ready to go any length 

Interestingly it does not detect the ip of router in the client but detects it in network tab and it also detects them in single mode when manually configured

**Comment by yubiuser** [comment]
NEMS is not a supported distribution. See here: https://docs.pi-hole.net/main/prerequisites/#software

Please use our Community help section over on discourse: https://discourse.pi-hole.net/c/bugs-problems-issues/community-help/36
